### DISCUSSION TITLE
How to Get NorthWind samples for a particular version

### DISCUSSION THREAD

**Discussion by zainvegeta** [serenity-is/Serenity#6245] [Tricks and Samples]
The template code have changed in the new versions. So if i want to refer an old version sample how can i get it?

**Comment by wezmag** [comment]
To get the full source code with Northwind samples, you can clone the repo with command below.
``` 
git clone --recurse-submodules https://github.com/serenity-is/Serene.git SereneDemo
```

If you want to refer to an old version sample, I think you need to create a project with old version of Serene.

**Comment by zainvegeta** [comment]
Yes i want to refer old version sample, can i install a perticular version of serene?

![image](https://user-images.githubusercontent.com/38037830/151541628-6427f358-329f-467f-8227-d1f33c40c3f6.png)

The screenshort shows the versions available, but seems the package is not updated in 3 years

**Reply by VictorTomaili** [reply]
The old version of the package was released 3 years ago. So why need to update a specific version again and again? We release new versions already. 

If you want a specific version of serene, you need to clone it with git and check out the related version of the commit that you want. Also if you need, you can create a template from the old version. But we recommend the latest version of the template from the visual studio marketplace.

**Comment by zainvegeta** [comment]
I did went through serene GitHub, but cant find any tags/releases. They are empty

But in the commits i can see the versions.
![image](https://user-images.githubusercontent.com/38037830/151652356-dbebe48a-a8d0-4f90-bec8-d5e2fe93f078.png)


You can close this thread. Thankyou
